.husky/prepare-commit-msg

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env sh
+. "$(dirname "$0")/_/husky.sh"
+
+COMMIT_MSG_FILE=$1
+
+BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
+
+BRANCH_NUMBER=$(echo "$BRANCH_NAME" | grep -o '[0-9]\+')
+
+if [ ! -z "$BRANCH_NUMBER" ];
+then
+  COMMIT_MSG=$(cat "$COMMIT_MSG_FILE")
+
+  MODIFIED_MSG=$(echo "$COMMIT_MSG" | sed "s/^\(.*: \)\(.*\)/\1$BRANCH_NUMBER - \2/")
+
+  COMMIT_REGEX="^(feat|fix|refactor|docs|chore|test): [0-9]{1,5} \-( [a-zA-Z-]+){2,}$"
```

Comment:
```suggestion
  COMMIT_REGEX="^(feat|fix|refactor|docs|chore|test): [0-9]{1,5} \-( [a-zA-Z ]+){2,}$"
```

I seppose here shuld be spase insted "-"?

Reply:
I don't think so because we have space before []. But we can have words with '-'.

Reply:
Yeah, I misread the pattern. Space is not needed here. But, then, shouldn't we add other symbols? ```'``` ```,``` ```;``` ```:``` ```"```

Reply:
I didn't think about them. Maybe something like this?
```suggestion
  COMMIT_REGEX="^(feat|fix|refactor|docs|chore|test): [0-9]{1,5} \-( [a-zA-Z-'"&]+[,:;]?){2,}$"
```

Reply:
I suppose, we can add [:print:]

Reply:
Then we can use `.*`, no?

Reply:
```suggestion
  COMMIT_REGEX="^(feat|fix|refactor|docs|chore|test): [0-9]{1,5} \- \D{3,}$"
```

Reply:
Maybe we leave only [ _-'&][:. ]  ?

Reply:
Something like this?
```suggestion
  COMMIT_REGEX="^(feat|fix|refactor|docs|chore|test): [0-9]{1,5} \-( [a-zA-Z_'&:.-]+){2,}$"
```

Reply:
Ok!

Reply:
Changed